src/main/java/cz/muni/fi/pa165/library/services/SingleLoanService.java

```diff
@@ -8,25 +8,51 @@
 
 /** @author Martin Páleník 359817
  *  based on http://zetcode.com/springboot/crudrepository/
+ *
+ *  This represents the business logic of the
+ *  loans of book or books by a member.
  */
 
 @Service
 public class SingleLoanService {
+
     private SingleLoanRepository singleLoanRepository;
 
+    /**
+     * Constructor creates single loan between a member and a book
```

Comment:
That is not true. The contructor only injects repository.

Reply:
True, corrected the JavaDoc based on [Java Practices](http://www.javapractices.com/topic/TopicAction.do?Id=60)